common/views/components/EventPromo/EventPromo.js

```diff
@@ -0,0 +1,156 @@
+// @flow
+import {Fragment} from 'react';
+import {spacing, font} from '../../../utils/classnames';
+import {isDatePast, formatDayDate, formatTime} from '../../../utils/format-date';
+import {UiImage} from '../Images/Images';
+import Icon from '../Icon/Icon';
+import type {EventPromo as Props} from '../../../model/events';
+
+function capitalize(word) {
+  return `${word.slice(0, 1).toUpperCase()}${word.slice(1).toLowerCase()}`;
+}
+
+function camelize(text, separator = '_') {
```

Comment:
Looks like we've already written this in `utils` (we could probably add `capitalize` as a util file as well

Reply:
I personally wouldn't as I sing these should be sparing as it should be left to content creators to do this, and when we use these programatically, it's a hack.

Reply:
well, now we've written a hack twice instead of only once!